client/account-status-settings/index.js

```diff
@@ -123,7 +123,13 @@ const AccountStatus = ( props ) => {
 	return (
 		<div>
 			<div>
-				<StatusChip accountStatus={ accountStatus.status } />
+				<StatusChip
+					accountStatus={ accountStatus.status }
+					poEnabled={ accountStatus.progressiveOnboarding.isEnabled }
```

Comment:
not sure where it's used by the client, @ismaeldcom maybe you know? I fixed the code accordingly, don't know how to test 🙈

Reply:
As we mentioned on Slack, it seems to be unreferenced code, so we can just remove it in a follow-up issue.

Reply:
Created follow up issue https://github.com/Automattic/woocommerce-payments/issues/6415